src/SwedbankPay.Sdk/Payments/CardPayments/CardPaymentAuthorizationRequestTransaction.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace SwedbankPay.Sdk.Payments.CardPayments
+{
+    public class CardPaymentAuthorizationRequestTransaction
+    {
+        protected internal CardPaymentAuthorizationRequestTransaction(string cardNumber,
```

Comment:
This class needs [JsonConstructor], it won't deserialize otherwise since it's protected internal.

Reply:
Fixed 👍